### DISCUSSION TITLE
How to handle registration from panel

### DISCUSSION THREAD

**Discussion by amoktar** [filamentphp/filament#11048] [Help] [answered]
### Package

Panel builder

### Package Version

v3.2.12

### How can we help you?

Hello guys , 

I'm having a panel with registration method chained
```php
return $panel
            ->id('dashboard')
            ->path('dashboard')
            ->login()
            ->registration()
```

How can I handle registration via this panel like setting some attributes to the users
In this case I need user registration from this panel to have role_slug attibute set to 'users' 

Thanks

**Comment by leandrocfe** [comment]
There is an example here that the profile is changed:
https://filamentphp.com/docs/3.x/panels/users#customizing-the-authentication-features

You can use the same approach

**Reply by leandrocfe** [reply, marked as answer]
If it works, mark as answered please

**Reply by amoktar** [reply]
Thanks @leandrocfe
